ci-operator/config/openshift-knative/serving/openshift-knative-serving-release-v1.5__411.yaml

```diff
@@ -392,33 +361,54 @@ tests:
           cpu: 100m
       timeout: 4h0m0s
     workflow: generic-claim
-- as: e2e-aws-ocp-411-continuous
+- as: test-e2e-tls-aws-ocp-411-continuous
   cluster_claim:
     architecture: amd64
     cloud: aws
     owner: openshift-ci
     product: ocp
     timeout: 1h0m0s
     version: "4.11"
-  cron: 0 17 * * 1-5
+  cron: 0 5 * * 2,6
```

Comment:
@pierDipi we might want to improve
`It is not currently possible to change the cron expression per job`
otherwise we'll have a lot of jobs at the same time :)

Reply:
Yes, discussed this a while ago, we can generate hours/days deterministically, wanna contribute the feature?

Reply:
I'll take a look 👍